@wmiler Test results of wmiler:styles -- All these were built and tested from source, and tested with several light and dark system themes.
* Debian 12 - OK
* Debian 13.2 - OK
* Ubuntu 25.04 - OK
* LUbuntu 24.04 - Works fine, but will not follow system theme like other distros do.  However, other apps don't either, so it's an LUbuntu OS problem, not a JS8Call problem.
* MX Linux - Works fine, but will not follow system theme like other distros do.  However, other apps do follow the system theme.
* Fedora 39 - OK
* Fedora 41 - OK
* Windows 11 24H2 - OK

In addition, I tested all these with a modified styles.h to add the mono font family to the currentFreq in the Linux section, line 283:
`" font-family: \"DejaVu Sans Mono\", \"Liberation Mono\", \"Noto Mono\", \"Ubuntu Mono\", monospace;"`

And that worked fine to provide the same monospaced font as the date/time text.

In all cases, the following warning shows during build, as I mentioned before in another place:
```
Building CXX object CMakeFiles/JS8Call.dir/JS8_UI/mainwindow.cpp.o
/home/jcounsil/development/JS8Call-improved/JS8_UI/mainwindow.cpp: In member function ‘void UI_Constructor::createStatusBar()’:
/home/jcounsil/development/JS8Call-improved/JS8_UI/mainwindow.cpp:1451:18: warning: variable ‘textColor’ set but not used [-Wunused-but-set-variable]
 1451 |     const QColor textColor = this->palette().color(QPalette::WindowText);
      |                  ^~~~~~~~~
[  5%] Linking CXX executable JS8Call
[100%] Built target JS8Call
```

73,
-Joe-
K0OG

--- a/JS8_UI/styles.h
+++ b/JS8_UI/styles.h
@@ -394,7 +394,7 @@ constexpr const char *MonitorButtonStyle =
 
 constexpr const char *LogQSOButtonStyle =
     "QPushButton {"
-    "    background-color:lightgray;"
+    "    background-color:#6699ff;"
     "    color:black;"
     "    padding:0.25em 0.25em; font-weight:normal;"
     "    border-style:solid;"
@@ -406,7 +406,19 @@ constexpr const char *LogQSOButtonStyle =
     "    color:black;"
     "}";
 
-constexpr const char *TuneButtonStyle = LogQSOButtonStyle; // Same as above
+constexpr const char *TuneButtonStyle =
+    "QPushButton {"
+    "    background-color:lightgray;"
+    "    color:black;"
+    "    padding:0.25em 0.25em; font-weight:normal;"
+    "    border-style:solid;"
+    "    border-width:0px;"
+    "    border-radius:6px;"
+    "}"
+    "QPushButton:checked {"
+    "    background-color:#6699ff;"
+    "    color:black;"
+    "}";
 
 constexpr const char *ModeButtonStyle =
     "QPushButton {"

--- a/JS8_UI/styles.h
+++ b/JS8_UI/styles.h
@@ -263,7 +263,7 @@ static inline QString logFrameStyle() {
                           " min-height: 40px;"
                           "}");
 #elif defined(Q_OS_WIN)
-    return QStringLiteral("QFrame#frame { background-color: #F2F2F0; }"
+    return QStringLiteral("QFrame#frame { background-color: #DDEEFF; }"
                           "QLabel#currentFreq {"
                           " color: #39FF14;"
                           " background-color: black;"
@@ -361,6 +361,7 @@ constexpr const char *ButtonGridStyle =
 constexpr const char *MonitorTxButtonStyle =
     "QPushButton {"
     "    background-color:lightgray;"
+    "    color:black;"
     "    padding:0.25em 0.25em; font-weight:normal;"
     "    border-style:solid;"
     "    border-width:0px;"
@@ -368,14 +369,17 @@ constexpr const char *MonitorTxButtonStyle =
     "}"
     "QPushButton:checked {"
     "    background-color:#22FF22;"
+    "    color:black;"
     "}"
     "QPushButton[transmitting=\"true\"] {"
     "    background-color:#FF2222;"
+    "    color:black;"
     "}";
 
 constexpr const char *MonitorButtonStyle =
     "QPushButton {"
     "    background-color:lightgray;"
+    "    color:black;"
     "    padding:0.25em 0.25em; font-weight:normal;"
     "    border-style:solid;"
     "    border-width:0px;"
@@ -383,11 +387,13 @@ constexpr const char *MonitorButtonStyle =
     "}"
     "QPushButton:checked {"
     "    background-color:#22FF22;"
+    "    color:black;"
     "}";
 
 constexpr const char *LogQSOButtonStyle =
     "QPushButton {"
     "    background-color:lightgray;"
+    "    color:black;"
     "    padding:0.25em 0.25em; font-weight:normal;"
     "    border-style:solid;"
     "    border-width:0px;"
@@ -395,6 +401,7 @@ constexpr const char *LogQSOButtonStyle =
     "}"
     "QPushButton:checked {"
     "    background-color:#6699ff;"
+    "    color:black;"
     "}";
 
 constexpr const char *TuneButtonStyle = LogQSOButtonStyle; // Same as above
@@ -406,14 +413,17 @@ constexpr const char *ModeButtonStyle =
     "    border-width:0px;"
     "    border-radius:6px;"
     "    background-color:#6699ff;"
+    "    color:black;"
     "}"
     "QPushButton:checked {"
     "    background-color:#6699ff;"
+    "    color:black;"
     "}";
 
 constexpr const char *SpotButtonStyle =
     "QPushButton {"
     "    background-color:lightgray;"
+    "    color:black;"
     "    padding:0.25em 0.25em; font-weight:normal;"
     "    border-style:solid;"
     "    border-width:0px;"
@@ -421,6 +431,7 @@ constexpr const char *SpotButtonStyle =
     "}"
     "QPushButton:checked {"
     "    background-color:#6699ff;"
+    "    color:black;"
     "}";
 
 }
@@ -428,7 +439,7 @@ constexpr const char *SpotButtonStyle =
 namespace Styles {
 
 constexpr const char *LogWidgetStyle =
-    "QFrame#logWidget { background-color: #F0F0F0; }";
+    "QFrame#logWidget { background-color: #DDEEFF; }";
 
 constexpr const char *DialFreqUpDownButtonStyle =
     "QPushButton {"
@@ -480,6 +491,7 @@ constexpr const char *ButtonGridStyle =
 constexpr const char *MonitorTxButtonStyle =
     "QPushButton {"
     "    background-color:lightgray;"
+    "    color:black;"
     "    padding:0.25em 0.25em; font-weight:normal;"
     "    border-style:solid;"
     "    border-width:0px;"
@@ -487,14 +499,17 @@ constexpr const char *MonitorTxButtonStyle =
     "}"
     "QPushButton:checked {"
     "    background-color:#22FF22;"
+    "    color:black;"
     "}"
     "QPushButton[transmitting=\"true\"] {"
     "    background-color:#FF2222;"
+    "    color:black;"
     "}";
 
 constexpr const char *MonitorButtonStyle =
     "QPushButton {"
     "    background-color:lightgray;"
+    "    color:black;"
     "    padding:0.25em 0.25em; font-weight:normal;"
     "    border-style:solid;"
     "    border-width:0px;"
@@ -502,11 +517,13 @@ constexpr const char *MonitorButtonStyle =
     "}"
     "QPushButton:checked {"
     "    background-color:#22FF22;"
+    "    color:black;"
     "}";
 
 constexpr const char *LogQSOButtonStyle =
     "QPushButton {"
     "    background-color:lightgray;"
+    "    color:black;"
     "    padding:0.25em 0.25em; font-weight:normal;"
     "    border-style:solid;"
     "    border-width:0px;"
@@ -514,6 +531,7 @@ constexpr const char *LogQSOButtonStyle =
     "}"
     "QPushButton:checked {"
     "    background-color:#6699ff;"
+    "    color:black;"
     "}";
 
 constexpr const char *TuneButtonStyle = LogQSOButtonStyle; // Same as above
@@ -525,14 +543,17 @@ constexpr const char *ModeButtonStyle =
     "    border-width:0px;"
     "    border-radius:4px;"
     "    background-color:#6699ff;"
+    "    color:black;"
     "}"
     "QPushButton:checked {"
     "    background-color:#6699ff;"
+    "    color:black;"
     "}";
 
 constexpr const char *SpotButtonStyle =
     "QPushButton {"
     "    background-color:lightgray;"
+    "    color:black;"
     "    padding:0.25em 0.25em; font-weight:normal;"
     "    border-style:solid;"
     "    border-width:0px;"
@@ -540,6 +561,7 @@ constexpr const char *SpotButtonStyle =
     "}"
     "QPushButton:checked {"
     "    background-color:#6699ff;"
+    "    color:black;"
     "}";
 
 }

--- a/JS8_UI/styles.h
+++ b/JS8_UI/styles.h
@@ -181,6 +181,8 @@ inline QString buttonStyle() {
             border: none;
             border-radius: 4px;
             padding: 3px 9px;
+            min-height: 15px;
+            max-height: 15px;
             font-family: "Segoe UI";
         }
         QPushButton:hover {

